src/Run/Executor.cs

```diff
@@ -25,24 +24,15 @@ public Executor()
         {
             SettingParameters = new Dictionary<string, string>();
             CommandParameters = new Dictionary<string, Dictionary<string, string>>();
-            string executorDirectory = Path.GetDirectoryName(typeof(Executor).GetTypeInfo().Assembly.Location);
-            configFilePath = Path.GetFullPath(Path.Combine(executorDirectory, @".."));
+            configFilePath = Directory.GetCurrentDirectory();
         }
 
-        public Executor(string configFile=null)
+        public Executor(string configFile) : this()
         {
-            SettingParameters = new Dictionary<string, string>();
-            CommandParameters = new Dictionary<string, Dictionary<string, string>>();
+            if (string.IsNullOrEmpty(configFile))
+                throw new ArgumentException("The configFile parameter cannot be null or empty.");
 
-            if (configFile == null)
-            {
-                string executorDirectory = Path.GetDirectoryName(typeof(Executor).GetTypeInfo().Assembly.Location);
-                configFilePath = Path.GetFullPath(Path.Combine(executorDirectory, @".."));
-            }
-            else
-            {
-                configFilePath = Path.GetDirectoryName(configFile);
-            }
+            configFilePath = Path.GetDirectoryName(configFile);
```

Comment:
Why wouldn't we actually accept the full config file path as opposed to hardcoding that the file must be called config.json